### ISSUE TITLE
Otterize network-mapper invalid URL escape

### ISSUE THREAD

**Issue by jonaszlasut** [otterize/network-mapper#162]
We're trying to use otterize network-mapper with rancher based cluster, which has cluster:server field in kubeconfig that looks like this

```
clusters:
- cluster:
    server: https://RANCHER_URL/k8s/clusters/CLUSTER_NAME
  name: CLUSTER_NAME
```

However when running `otterize network-mapper list` we're getting the below error

```
Error: error upgrading connection: error creating request: parse "https://<RANCHER_URL>%2Fk8s%2Fclusters%2FCLUSTER_NAME/api/v1/namespaces/otterize-system/pods/otterize-network-mapper-65b5b9c654-tkgvq/portforward": invalid URL escape "%2F"
```

**Comment by jonaszlasut** [comment]
We use cli in version 1.0.1

otterize version -> `Version: v1.0.1`

**Comment by orishoshan** [comment]
Hi @jonaszlasut, looking at this now, and seem to be unable to reproduce with the same configuration, however I have an idea of what might be the cause anyway. Can I ask you to join us on Slack at https://joinslack.otterize.com and have you try a binary from a branch?

**Comment by jonaszlasut** [comment]
Hi @orishoshan , I've joined the slack. How can I contact you?

**Comment by jonaszlasut** [comment]
H @orishoshan I've sent you a slack message

**Comment by orishoshan** [comment]
It worked for @jonaszlasut - published in v1.0.2.
